frontend/src/main/components/Leaderboard/LeaderboardTable.js

```diff
@@ -4,6 +4,11 @@ import { hasRole } from "main/utils/currentUser";
 // should take in a players list from a commons
 export default function LeaderboardTable({ leaderboardUsers , currentUser }) {
 
+    let USD = new Intl.NumberFormat("en-US", {
```

Comment:
any particular reason to use `let` here vs. using `const` ?

According to my sources (e.g. the one below) `const` is preferred for all js variables unless we expect the variable's value to change, which in this case we don't, I think.

https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

Reply:
That's on me. I confused myself on how let vs. const worked. Fixed!